src/server/requestHandlers.js

```diff
@@ -41,9 +49,16 @@ sjrk.storyTelling.server.handleBrowseStories = function (request, viewDatasource
             message: errorAsJSON
         });
     });
-
 };
 
+/**
+ * Extracts a collection of stories from the raw CouchDB Browse response and
+ * returns an object with that collection along with some statistics about it
+ *
+ * @param {Object} response - the raw Browse response
+ *
+ * @return {Object} - a collection of stories as well as the total number and page offset

```

Comment:
Use @typedef to describe the object structure for important parameters or returns. If the object structure is too dynamic to use @typedef, an example of the object would be helpful too. Thanks.